packages/slice-machine/src/hooks/useSMTracker.tsx

```diff
@@ -41,8 +41,7 @@ const useSMTracker = () => {
 
   // Handles if the user login/logout outside of the app.
   useEffect(() => {
-    if (shortId && intercomHash)
-      void Tracker.get().identifyUser(shortId, intercomHash);
+    if (shortId && intercomHash) void Tracker.get().identifyUser();
```

Comment:
In this hook, changes to short id and intercomHash trigger the event. To sure if they can go rom being defined to undefined, I'll look into it

Reply:
The hook be called on the first render, and again if shortId or intercomHash change, right?

Reply:
Right, so this is used only for people that comes back to slice machine being logged in but our backend set the shortId and intercomHash.

Gosh this is such a edge case, i'm not even sure it would be useful anymore.

Reply:
Well that and if the condition is removed the shortId and intercomHash will fire... which actually might be what we want.